dist/angular-swaggerific.js

```diff
@@ -115,6 +115,12 @@
                 });
             });
 
+			self.api["cancelRequests"] = function() {
```

Comment:
Since we are returning the `self.api` in the constructor, I can see why you added `cancelRequests` as a property of `api`. However, if the user has a method named `cancelRequests` defined in their Swagger API, this will override it. As an alternative, we could add a static method to `AngularSwaggerific` which takes `api` as a parameter? 

`AngularSwaggerific.cancelRequests = function(api) { }`

This way we can be sure we aren't polluting the API namespace.